app/brave_settings_strings.grdp

```diff
@@ -1443,6 +1443,17 @@ Leo. This action cannot be undone.
     Default model for new conversations
   </message>
 
+  <!-- Settings / Playlist -->
+  <message name="IDS_SETTINGS_PLAYLIST_ENABLE_PLAYLIST_LABEL" desc="The text for setting to en/disable Playlist feature">
+    Enable Playlist
+  </message>
+  <message name="IDS_SETTINGS_PLAYLIST_CACHE_BY_DEFAULT_LABEL" desc="The text for setting to en/disable media cache behavior">
+    Automatically add files for offline viewing
+  </message>
+  <message name="IDS_SETTINGS_PLAYLIST_CACHE_BY_DEFAULT_SUB_LABEL" desc="A description for setting to en/disable media cache behavior">
+    This option will automatically cache media files so you can play without an Internet connection.
```

Comment:
"play *them*" sounds a bit better to me.

Reply:
Great! done.